Index: shell/rb-shell-player.c
===================================================================
RCS file: /cvs/gnome/rhythmbox/shell/rb-shell-player.c,v
retrieving revision 1.71
diff -u -r1.71 rb-shell-player.c
--- shell/rb-shell-player.c	17 Aug 2003 04:17:28 -0000	1.71
+++ shell/rb-shell-player.c	17 Aug 2003 20:39:17 -0000
@@ -1720,7 +1720,8 @@
 			root = gdk_screen_get_root_window (screen);
 
 			for (j = 0; j < 4 ; j++) {
-				grab_mmkey (keycodes[j], root);
+				if (keycodes[j] != 0)
+					grab_mmkey (keycodes[j], root);
 			}
 
 			gdk_window_add_filter (root, filter_mmkeys,
