--- rhythmbox-0.11.6/data/rhythmbox.desktop.in.in.wiget	2008-07-02 10:51:58.000000000 +0200
+++ rhythmbox-0.11.6/data/rhythmbox.desktop.in.in	2008-08-30 13:30:55.000000000 +0200
@@ -7,7 +7,7 @@ Terminal=false
 Type=Application
 Icon=rhythmbox
 X-GNOME-DocPath=rhythmbox/rhythmbox.xml
-Categories=GNOME;GTK;AudioVideo;
+Categories=GNOME;GTK;AudioVideo;Audio;Player;
 MimeType=application/x-ogg;application/ogg;audio/x-scpls;audio/x-mp3;audio/x-mpeg;audio/mpeg;audio/x-mpegurl;audio/x-flac;x-content/audio-cdda;x-content/audio-player;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
