diff -aurN rhythmbox-0.9.4.90.orig/widgets/eggtrayicon.c rhythmbox-0.9.4.90/widgets/eggtrayicon.c
--- rhythmbox-0.9.4.90.orig/widgets/eggtrayicon.c	2006-05-19 10:43:54.000000000 +0200
+++ rhythmbox-0.9.4.90/widgets/eggtrayicon.c	2006-06-15 12:48:33.706771752 +0200
@@ -548,39 +548,35 @@
     {
       secondary = "";
     }
-
+  
+  pixbuf = NULL;
+  
   esc_primary = g_markup_escape_text (primary, strlen (primary));
   esc_secondary = g_markup_escape_text (secondary, strlen (secondary));
-  icon->notify->handle = notify_notification_new (esc_primary,
-                                                  esc_secondary,
-                                                  NULL,
-                                                  GTK_WIDGET (icon));
-  g_free (esc_primary);
-  g_free (esc_secondary);
-
-  notify_notification_set_timeout (icon->notify->handle, timeout);
 
   if (msgicon)
     {
+      icon->notify->handle = notify_notification_new (esc_primary,
+                                                      esc_secondary,
+                                                      NULL,
+                                                      GTK_WIDGET (icon));
       pixbuf = gtk_image_get_pixbuf (GTK_IMAGE (msgicon));
     }
   else
     {
-      GtkIconTheme *theme;
-      gint icon_size;
-
-      theme = gtk_icon_theme_get_default ();
-      gtk_icon_size_lookup (GTK_ICON_SIZE_DIALOG, &icon_size, NULL);
-      pixbuf = gtk_icon_theme_load_icon (theme,
-                                         "gnome-media-player",
-                                         icon_size,
-                                         0,
-                                         NULL);
+      icon->notify->handle = notify_notification_new (esc_primary,
+                                                      esc_secondary,
+                                                      "gnome-media-player",
+                                                      GTK_WIDGET (icon));
     }
-
+    
+  notify_notification_set_timeout (icon->notify->handle, timeout);
+  g_free (esc_primary);
+  g_free (esc_secondary);
+  
   if (pixbuf)
     {
-#if (LIBNOTIFY_VERSION_MAJOR == 0 && LIBNOTIFY_VERSION_MINOR <=3 && LIBNOTIFY_VERSION_MICRO < 2)
+#if (LIBNOTIFY_VERSION_MICRO < 2) && (LIBNOTIFY_VERSION_MINOR < 4)
       notify_notification_set_icon_data_from_pixbuf (icon->notify->handle, pixbuf);
 #else
       notify_notification_set_icon_from_pixbuf (icon->notify->handle, pixbuf);
