diff -urN rhythmbox-0.9.8/corelib/Makefile.am rhythmbox-0.9.8.new/corelib/Makefile.am
--- rhythmbox-0.9.8/corelib/Makefile.am	2007-01-02 02:56:23.000000000 +0100
+++ rhythmbox-0.9.8.new/corelib/Makefile.am	2007-02-21 17:58:12.000000000 +0100
@@ -19,7 +19,9 @@
 	$(top_builddir)/metadata/librbmetadata.la	\
 	$(top_builddir)/widgets/librbwidgets.la         \
 	$(top_builddir)/backends/librbbackends.la	\
-	$(top_builddir)/lib/librb.la
+	$(top_builddir)/lib/librb.la			\
+	$(NOTIFY_LIBS)
+
 
 librhythmbox_core_la_LDFLAGS = -export-dynamic
 
diff -urN rhythmbox-0.9.8/plugins/daap/Makefile.am rhythmbox-0.9.8.new/plugins/daap/Makefile.am
--- rhythmbox-0.9.8/plugins/daap/Makefile.am	2007-02-01 08:13:57.000000000 +0100
+++ rhythmbox-0.9.8.new/plugins/daap/Makefile.am	2007-02-21 17:57:51.000000000 +0100
@@ -42,7 +42,8 @@
 libdaap_la_LDFLAGS = $(PLUGIN_LIBTOOL_FLAGS)
 libdaap_la_LIBADD = 					\
 	$(DBUS_LIBS)					\
-	$(MDNS_LIBS)
+	$(MDNS_LIBS)					\
+	$(GSTREAMER_0_10_LIBS)
 
 INCLUDES = 						\
         -DGNOMELOCALEDIR=\""$(datadir)/locale"\"        \
