diff -urN rhythmbox-0.9.8/configure.ac rhythmbox-0.9.8.new/configure.ac
--- rhythmbox-0.9.8/configure.ac	2007-02-24 12:16:55.000000000 +0100
+++ rhythmbox-0.9.8.new/configure.ac	2007-02-24 12:59:25.000000000 +0100
@@ -910,7 +910,7 @@
 
 
 dnl check, for unit tests
-AM_PATH_CHECK([], have_check=yes, have_check=no)
+PKG_CHECK_MODULES(CHECK, check >= 0.9.4, have_check=yes, have_check=no)
 AM_CONDITIONAL([HAVE_CHECK],[test "x$have_check" = "xyes"])
 
 
