diff -aurN rhythmbox-0.9.3.orig/data/rhythmbox.desktop.in rhythmbox-0.9.3/data/rhythmbox.desktop.in
--- rhythmbox-0.9.3.orig/data/rhythmbox.desktop.in	2006-01-24 07:51:58.000000000 +0100
+++ rhythmbox-0.9.3/data/rhythmbox.desktop.in	2006-02-02 14:29:52.744276000 +0100
@@ -8,7 +8,7 @@
 Type=Application
 Icon=rhythmbox.png
 X-GNOME-DocPath=rhythmbox/rhythmbox.xml
-Categories=GNOME;GTK;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;AudioVideo;Audio;Player;
 MimeType=application/x-ogg;application/ogg;audio/x-mp3;audio/x-scpls;audio/x-mp3;audio/x-mpeg;audio/mpeg;audio/x-mpegurl;application/x-flac
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
