diff -aurN rhythmbox-0.9.1.orig/configure.ac rhythmbox-0.9.1/configure.ac
--- rhythmbox-0.9.1.orig/configure.ac	2005-10-16 11:50:21.000000000 +0200
+++ rhythmbox-0.9.1/configure.ac	2005-10-17 11:03:13.000000000 +0200
@@ -401,7 +401,7 @@
 AC_SUBST(RHYTHMBOX_CFLAGS)
 AC_SUBST(RHYTHMBOX_LIBS)
 
-ALL_LINGUAS="am ar az be bg ca cs cy da de el en_CA en_GB es et eu fi fr ga gl he hr hu id is it ja ko lt lv mk ml mn ms nb ne nl no pa pl pt pt_BR ro ru rw sk sr sr@Latn sv th tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="am az be bg ca cs cy da de el en_CA en_GB es et eu fi fr ga gl he hr hu id is it ja ko lt lv mk ml ms nb ne nl no pa pl pt pt_BR ro ru rw sk sr sr@Latn sv th tr uk vi zh_CN zh_TW"
 
 GETTEXT_PACKAGE=rhythmbox
 AC_SUBST(GETTEXT_PACKAGE)
