diff -urN rhythmbox-0.7.1.orig/configure.ac rhythmbox-0.7.1/configure.ac
--- rhythmbox-0.7.1.orig/configure.ac	2004-03-17 07:33:42.169961136 +0100
+++ rhythmbox-0.7.1/configure.ac	2004-03-17 07:34:02.657846504 +0100
@@ -454,7 +454,7 @@
 AC_SUBST(RHYTHMBOX_CFLAGS)
 AC_SUBST(RHYTHMBOX_LIBS)
 
-ALL_LINGUAS="am ar az be bg ca cs cy da de el es et fi fr ga hr hu id it ja ko lt lv mn ms nl no pl pt pt_BR ro ru sr sr@Latn sv tr uk vi zh_TW"
+ALL_LINGUAS="am ar az be bg ca cs cy da de el es et fi fr ga hr hu id it ja ko lt lv mn ms nb nl pl pt pt_BR ro ru sr sr@Latn sv tr uk vi zh_TW"
 
 GETTEXT_PACKAGE=rhythmbox
 AC_SUBST(GETTEXT_PACKAGE)
