diff -aurN rhythmbox-0.8.5.orig/data/rhythmbox.desktop.in rhythmbox-0.8.5/data/rhythmbox.desktop.in
--- rhythmbox-0.8.5.orig/data/rhythmbox.desktop.in	2004-06-21 17:57:18.000000000 +0200
+++ rhythmbox-0.8.5/data/rhythmbox.desktop.in	2004-09-17 07:29:25.730254608 +0200
@@ -7,7 +7,7 @@
 Type=Application
 Icon=rhythmbox.png
 X-GNOME-DocPath=rhythmbox/rhythmbox.xml
-Categories=GNOME;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;Audio;Player;
 MimeType=application/x-ogg;application/ogg;audio/x-mp3;audio/x-scpls;audio/x-mp3;audio/x-mpeg;audio/mpeg;audio/x-mpegurl;application/x-flac
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
