diff -urN aa/rhythmbox-0.7.0/configure.ac rhythmbox-0.7.0/configure.ac
--- aa/rhythmbox-0.7.0/configure.ac	2004-02-05 17:05:10.000000000 +0100
+++ rhythmbox-0.7.0/configure.ac	2004-03-10 17:59:59.731181480 +0100
@@ -270,9 +270,9 @@
 				[don't build with Ogg Vorbis support]))
 if test x"$with_metadata" = xmonkeymedia; then
 if test x"$enable_vorbis" != xno; then
-  MM_PATH_OGG(enable_vorbis=yes,enable_vorbis=no)
+  XIPH_PATH_OGG(enable_vorbis=yes,enable_vorbis=no)
   if test x"$enable_vorbis" = xyes; then
-    MM_PATH_VORBIS(,enable_vorbis=no)
+    XIPH_PATH_VORBIS(,enable_vorbis=no)
   fi
   if test x"$enable_vorbis" = xyes; then
     if test x"$have_gstreamer" = xyes; then
