--- trunk/bindings/python/rb.override	2007/10/23 22:27:34	5412
+++ trunk/bindings/python/rb.override	2007/10/24 22:53:08	5414
@@ -1313,6 +1313,7 @@
 static PyObject *
 _wrap_rb_removable_media_manager_queue_transfer(PyGObject *self, PyObject *args, PyObject *kwargs)
 {
+#ifdef ENABLE_TRACK_TRANSFER
 	static char *kwlist[] = { "entry", "dest", "mimetypes", "callback", "data", NULL };
 	PyObject *py_entry, *py_mimes;
 	char *dest = NULL;
@@ -1342,5 +1343,9 @@
 	g_list_free (mimes);
 	Py_INCREF(Py_None);
 	return Py_None;
+#else
+	PyErr_SetString(PyExc_NotImplementedError, "Track transfer is disabled in this build of Rhythmbox");
+	return NULL;
+#endif
 }
 
