diff -urN aa/rhythmbox-0.7.0/configure.ac rhythmbox-0.7.0/configure.ac
--- aa/rhythmbox-0.7.0/configure.ac	2004-02-05 17:05:10.000000000 +0100
+++ rhythmbox-0.7.0/configure.ac	2004-03-10 17:48:09.806106496 +0100
@@ -122,7 +122,7 @@
 	HAVE_SOUNDSYSTEM=no
 
 	dnl try 0.7, in the future also 0.8
-	GST_MAJORMINOR=0.7
+	GST_MAJORMINOR=0.8
 	PKG_CHECK_MODULES(SOUNDSYSTEM, \
 		gstreamer-$GST_MAJORMINOR gstreamer-libs-$GST_MAJORMINOR
 		gstreamer-control-$GST_MAJORMINOR gstreamer-gconf-$GST_MAJORMINOR,
